spec/rails_helper.rb

```diff
@@ -17,7 +17,8 @@
     app,
     inspector: ENV["DEBUG_TESTS"],
     js_errors: true,
-    window_size: [1200, 800]
+    window_size: [1200, 800],
+    process_timeout: 30
```

Comment:
This is the timeout error generated by the ferrum when it was unable to start the Chrome process. 

<img width="1218" alt="image" src="https://github.com/nhsuk/manage-vaccinations-in-schools/assets/15608/04ebaf86-2194-498f-8e6e-a021ce2a22ce">

I'm a little suspicious of this fix because it was the same test that had this issue both times, and I would've expected this error to be triggered randomly by different tests. I only tried re-running the job twice, and both times it failed it took 2m30s (vs <1m normally) so it's possible something else was going on on GitHub's end.

Reply:
After chat with Theo, this test actually uses JS so does try to load up the browser, which is very likely what's triggering this issue. Adjusting the `process_timeout` here does potentially make sense, and as this does happen around 5% of the time we'll try to increase it and see if it helps.